### ISSUE TITLE
FEATURE request: export menu

### ISSUE THREAD

**Issue by thecyrus** [TheGameCreators/GameGuruRepo#91]
I really think it would be a GREAT idea to make a standalone build menu just like the one FPSC had, to edit settings, menu/loading screens and more, instead of modifying everything into folders for each project.

**Comment by Granada1** [comment]
That’s a good request

**Comment by Teabone3** [comment]
That's something a lot of us have been asking for over the years with Game Guru. There is however StyleGuru a 3rd party program to help support with this but i sure do miss the ease of use with FPSC's export settings.

**Comment by thecyrus** [comment]
Yes, that's why I wanted to put it here too.
I think it's as simple as needed.

**Comment by Belidos3D** [comment]
Yes please, i would love to be able to have a menu when creating standalone that allows you to choose various options, for example which titles set to use, what screen resolutions to allow and so on.
